### PR TITLE
Replace delegate by call_operation when mapping abstract components

### PR DESCRIPTION
**Pull request by BertrandD**

### CODE REVIEW

File: alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/modifiers/AbstractMappingModifier.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    public static final String YORC_WF_ABSTRACT_MAPPING_MODIFIER_TAG = "yorc-wf-abstract-mapping-modifier";
+
+    @Override
+    public void process(Topology topology, FlowExecutionContext context) {
+        safe((Map<String, NodeTemplate>)context.getExecutionCache().get(FlowExecutionContext.MATCHING_ORIGINAL_NODES)).forEach((name, nodeTemplate) -> handleNodeTemplate(context, topology, nodeTemplate));
+    }
+
+    private void handleNodeTemplate(FlowExecutionContext context, Topology topology, NodeTemplate originalNodeTemplate) {
+        if(!isCompute(originalNodeTemplate) && !isNetwork(originalNodeTemplate) && ToscaContext.getOrFail(NodeType.class, originalNodeTemplate.getType()).isAbstract()) {
```
Comment (loicalbertin) [inline alien4cloud-yorc-plugin/src/main/java/org/ystia/yorc/alien4cloud/plugin/modifiers/AbstractMappingModifier.java]: BlockStorages should also be preserved as delegate

**Comment by loicalbertin** [comment]
We need to talk to see if this is the good way to do it.

**Comment by loicalbertin** [comment]
README update is missing

**Comment by adidanes** [comment]
The corresponding Issue (https://github.com/ystia/yorc-a4c-plugin/issues/35) was closed.
It won't be fixed for the moment

**Pull request closed without merging**